### ISSUE TITLE
liquibase-hibernate6-4.25.0.jar contains classes from other libraries

### ISSUE THREAD

**Issue by haster** [liquibase/liquibase-hibernate#631]
[liquibase-hibernate6-4.25.0.jar](https://github.com/liquibase/liquibase-hibernate/releases/download/v4.25.0/liquibase-hibernate6-4.25.0.jar) seems to have incorporated all class files from its dependencies into the jar. This includes slf4j, jakarta-activation and hibernate.

This makes the jar unnecessary "fat" (compare  hibernate6-4.25.0.jar  at 17.5 MB and hibernate6-4.24.0.jar at 70 KB) but more importantly, this can lead to problems for users of the hibernate6 jar if the incorporated classfiles are from a different version than the one they are using themselves, since they end up with different versions of the same class on the classpath which can lead to any number of subtle but potentially huge problems.

Please ensure the jars are build "thin", not "fat".

**Comment by jsteenbeeke** [comment]
The same goes for [liquibase-hibernate5-4.25.0.jar](https://github.com/liquibase/liquibase-hibernate/releases/download/v4.25.0-hibernate5/liquibase-hibernate5-4.25.0.jar)

**Comment by jsteenbeeke** [comment]
This appears to be fixed in 4.25.1

**Comment by haster** [comment]
There does not seem to be a liquibase-hibernate6-4.25.1 yet, but as one assumes the fix will be trivially similar this is good news nevertheless.

**Comment by sakonn** [comment]
I am trying to use version `4.25.1` and also getting a warning message because of `slf4j-simple` dependency. I had to explicitly remove that dependency to get rid of the message (gradle):
```
implementation("org.liquibase.ext:liquibase-hibernate6:4.25.1") {
    // Remove duplicate logging binding: https://www.slf4j.org/codes.html#multiple_bindings
    exclude (group = "org.slf4j", module = "slf4j-simple")
}
```

**Comment by filipelautert** [comment]
hi!
this is going to be fully fixed on 4.27.0 that will be released later this week.

**Comment by filipelautert** [comment]
We just released 4.27.0 and 4.26.0 that failed last time -> https://central.sonatype.com/artifact/org.liquibase.ext/liquibase-hibernate5/versions

They must be reaching other mirrors  in the next hours, could you confirm that they fix this issue?
